Makefile

```diff
@@ -2,10 +2,15 @@
 
 up:
 	minikube start
+	minikube addons enable ingress
+	minikube addons enable dashboard
 	kubectl create namespace gitlab
 	helm repo add gitlab https://charts.gitlab.io
 	helm repo update
-	helm install gitlab gitlab/gitlab --namespace gitlab --values values.yaml --version v7.1.0
```

Comment:
When we should use the install command and when to use the upgrade command?

Reply:
We can use both, but according to the best practices in this [article](https://codersociety.com/blog/articles/helm-best-practices) - rule #13 it's better to use upgrade with `--install` flag `Always use the helm upgrade --install command. It installs the charts if they are not already installed. If they are already installed, it upgrades them.`

Reply:
Thanks for sharing this info.   Not sure if the upgrade process is aligned with the reproducible build practice. Every external dependency is supposed to be pinned versioned.